src/Test.FunctionalTests.BinSkim.Driver/AnalyzeCommandTest.cs

```diff
@@ -1,15 +1,77 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.BinaryParsers;
+using Microsoft.CodeAnalysis.IL;
+using Microsoft.CodeAnalysis.Sarif;
+
+using Moq;
+
+using Newtonsoft.Json;
+
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.BinSkim.Driver
 {
     public class AnalyzeCommandTest
```

Comment:
>AnalyzeCommandTest [](http://example.com/codeflow?start=17&length=18)

AnalyzeCommandTests? i.e., add the 's', change the file name.

Reply:
done!